qir/cli-tests/test_profile_cli_config.py

```diff
@@ -0,0 +1,114 @@
+import subprocess
+import sys
+
+TESTS = {
+    "provider_4bf9": [
```

Comment:
May be it is possible to use a more sensible fragment instead of `4bf9`?
Otherwise how can the reader/user memorize it and make sure that `4bf9` is the right fragment to use here?